python/tests/cli/test_console.py

```diff
@@ -189,13 +189,17 @@ def test_describe_table(_):
                         2 ASC NULLS FIRST
                         bucket[4](3) DESC NULLS LAST
                       ]
-Schema                Schema
+Current schema        Schema, id=1
                       ├── 1: x: required long
                       ├── 2: y: required long (comment)
                       └── 3: z: required long
+Current snapshot      id=3055729675574597004, parent_id=3051729675574597004,
+                      schema_id=1
```

Comment:
What caused this line wrap? I don't see it in the code above.

Reply:
`rich` automatically overflows to a newline (with the correct indentation, that's why `schema_id` is nicely aligned with `id`. In a normal terminal, it will infer the size. There are multiple options to control the overflow: https://rich.readthedocs.io/en/stable/console.html#overflow In the terminal it looks nicer if you have sufficient width.